docs/05-technical-reference/00-custom-resources/telemetry-01-logpipeline.md

```diff
@@ -0,0 +1,127 @@
+  - lastTransitionTime: "2022-11-25T12:39:26Z"
+    reason: FluentBitDaemonSetRestartCompleted
+    type: Running
+```
+
+For further LogPipeline examples, see the [samples](https://github.com/kyma-project/kyma/blob/main/components/telemetry-operator/config/samples) directory.
+
+## Custom resource parameters
+
+### spec attribute
+
+For details, see the [LogPipeline specification file](https://github.com/kyma-project/kyma/blob/main/components/telemetry-operator/apis/telemetry/v1alpha1/logpipeline_types.go).
+
+| Parameter | Type | Description |
+|---|---|---|
+| input | object | Definition where to collect logs, including selector mechanisms. |
+| input.application | object | Input type for application logs collection. |
+| input.application.namespaces | object | Provides selectors for Namespaces. Selectors are mutually exclusive. |
+| input.application.namespaces.include | []string | List of Namespaces from which logs are collected. |
+| input.application.namespaces.exclude | []string | List of Namespaces to exclude during log collection from all Namespaces. |
+| input.application.namespaces.system | boolean | Set to `true` if collecting from all Namespaces should include system Namespaces as well. |
+| input.application.containers | []string | Provides selectors for containers. Selectors are mutually exclusive. |
+| input.application.containers.include | []string | List of containers to collect from. |
+| input.application.containers.exclude | []string | List of containers to exclude. |
+| input.application.keepAnnotations | boolean | Indicates whether to keep all Kubernetes annotations. Default is `false`. |
+| input.application.dropLabels | boolean | Indicates whether to drop all Kubernetes labels. Default is `false`. |
+| filters | []object | List of [Fluent Bit filters](https://docs.fluentbit.io/manual/pipeline/filters) to apply to the logs processed by the pipeline. Filters are executed in sequence, as defined. They are executed before logs are buffered, and with that, are not executed on retries.|
+| filters[].custom | string | Filter definition in the Fluent Bit syntax. Note: If you use a `custom` output, you put the LogPipeline in [unsupported mode](./../../01-overview/main-areas/telemetry/telemetry-02-logs.md#unsupported-mode).|
+| output | object | [Fluent Bit output](https://docs.fluentbit.io/manual/pipeline/outputs) where you want to push the logs. Only one output can be specified. |
+| output.grafana-loki | object | [Fluent Bit grafana-loki output](https://grafana.com/docs/loki/v2.2.x/clients/fluentbit/). Note: This output is considered legacy and is only provided for backwards compatibility with the [deprecated](https://kyma-project.io/blog/2022/11/2/loki-deprecation/) in-cluster Loki instance. It might not be compatible with latest Loki versions. For integration with a custom Loki installation use the `custom` output with name `loki` instead, see also [this tutorial](https://github.com/kyma-project/examples/tree/main/loki). |
```

Comment:
```suggestion
| output.grafana-loki | object | [Fluent Bit grafana-loki output](https://grafana.com/docs/loki/v2.2.x/clients/fluentbit/). **Note:** This output is considered legacy and is only provided for backward compatibility with the [deprecated](https://kyma-project.io/blog/2022/11/2/loki-deprecation/) in-cluster Loki instance. It might not be compatible with the latest Loki versions. For integration with a custom Loki installation use the `custom` output with the name `loki` instead, see also [this tutorial](https://github.com/kyma-project/examples/tree/main/loki). |
```